corehq/apps/app_manager/suite_xml/features/mobile_ucr.py

```diff
@@ -30,9 +31,21 @@ class ReportModuleSuiteHelper(object):
     def __init__(self, report_module):
         assert isinstance(report_module, ReportModule)
         self.report_module = report_module
-        self.domain = self.report_module.get_app().domain
+        self.domain = self.app.domain
         self._loaded = None
 
+    @cached_property
+    def app(self):
+        return self.report_module.get_app()
+
+    @cached_property
```

Comment:
is caching necessary here? I don't think these (`mobile_ucr_restore_version` and `app`) are expensive in any way

Reply:
I've conditioned myself to view methods like `get_app()` to be expensive, but it appears to already be loaded. Will fix